content/docs/other-guides/usage-reporting.md

```diff
@@ -40,13 +49,24 @@ kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
         name: manifests
         path: common/spartakus
     name: spartakus
-```
+  ```
 
-- Delete the entry in KfDef.Spec.ComponentParams for spartakus
+- Delete the Spartakus entry in the `componentParams` section. These are the
```

Comment:
You can delete this section about componentParams. componentParams is the old way of doing things. Given the instructions above about kfDef.Spec.applications; they should just remove it from KfDef.Spec.Applications.

Reply:
Done.